### ISSUE TITLE
Add a new ^ls command that only gives single multiplier (as opposed to square multipliers), like ^lss

### ISSUE THREAD

**Issue by muffin-rice** [TsubakiBotPad/padbot-cogs#91]

**Comment by RheingoldRiver** [comment]
Command would be `^lssingle` with alias `^lss`

So `^lss Tsubaki` would return:
```
Multiplier [1/12.5/1 Resist 25%]
No. 3260 Yamato Flame Dragon Caller, Tsubaki
2.5x ATK and reduce damage taken by 25% when matching Fire and Light; 5x ATK for matched Att. when matching 5 Orbs with 1+ enhanced
```

**Comment by RheingoldRiver** [comment]
closed by #157
